src/devices/OneWire/FamilyBindings/OneWireThermometerDevice.cs

```diff
@@ -15,23 +14,50 @@ namespace Iot.Device.OneWire
     public partial class OneWireThermometerDevice : OneWireDevice
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="OneWireThermometerDevice"/> class
+        /// Initializes a new instance of the <see cref="OneWireThermometerDevice"/> class.
         /// </summary>
-        /// <param name="bus">The 1-wire bus the device is found on</param>
-        /// <param name="deviceId">The id of the device</param>
-        /// <param name="family">The 1-wire fmily id</param>
-        protected internal OneWireThermometerDevice(OneWireBus bus, string deviceId, DeviceFamily family)
-            : base(bus, deviceId, family)
+        /// <param name="busId">The 1-wire bus the device is found on.</param>
+        /// <param name="devId">The id of the device.</param>
+        public OneWireThermometerDevice(string busId, string devId)
+            : base(busId, devId)
         {
         }
 
         /// <summary>
-        /// Enumerate all devices in system of type thermometer
+        /// Check if family is compatible with this type of devices.
+        /// </summary>
+        /// <param name="family">The family to check.</param>
+        /// <returns>Returns true if device is compatible.</returns>
+        public static bool IsCompatible(DeviceFamily family)
+        {
+            return family == DeviceFamily.Ds18S20 ||
+                   family == DeviceFamily.Ds18B20 ||
+                   family == DeviceFamily.Ds1825 ||
+                   family == DeviceFamily.Ds28EA00;
```

Comment:
`|| family == DeviceFamily.Thermometer`?

Reply:
That is a made up family id for filtering. It is not a valid id, no device should be created with it.

Reply:
One solution is to create two enum, add DeviceFamilyFilter. But one is a superset so seems overkill.

Reply:
I'd make this method private/internal. I'm not sure what it's for (from user perspective)